lib/winapi/winbase.h

```diff
@@ -45,6 +45,14 @@ typedef struct _INIT_ONCE
 #define INIT_ONCE_INIT_FAILED 0x00000004UL
 typedef BOOL CALLBACK PINIT_ONCE_FN(PINIT_ONCE, PVOID, PVOID *);
 
+typedef struct _CONDITION_VARIABLE {
+    INIT_ONCE initOnce;
+    PVOID eventHandles[2];
+    _Atomic int waitCount;
```

Comment:
Is this struct based on the MS internals? If yes, then is this keyword compatible with how MS does this? I worry about weird warnings / errors, if we ever pass this to functions which weren't written with C11 in mind.